### PR TITLE
docs(Accessibility): improve acc descriptions for components

### PR DESCRIPTION
**Pull request by jurokapsiar**
Improve acc descriptions for components

### CODE REVIEW

File: packages/react/src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -275,5 +275,7 @@ Dialog.slotClassNames = {
 
 /**
  * A Dialog indicates a possible user action.
```
Comment (hughreeling) [inline packages/react/src/components/Dialog/Dialog.tsx]: I don't think that's good description of dialog.  We can look at other frameworks for better description, but perhaps something like, a dialog is used to interrupt the standard flow to get some particular input from the user.

Comment (jurokapsiar) [reply packages/react/src/components/Dialog/Dialog.tsx]: it is not - I have missed this one :)

File: package.json [hunk 2]
```diff
@@ -112,6 +112,7 @@
     "react-dom": "^16.8.6",
     "react-element-to-jsx-string": "^14.0.2",
     "react-hot-loader": "^4.8.2",
+    "react-markdown": "^4.0.8",
```
Comment (bmdalex) [inline package.json:115]: @layershifter @kuzhelov @miroslavstastny 
- are we ok with adding this dependency?
- can we suggest some existing tooling to achieve this?

**Review by bmdalex — COMMENTED** [review]
pls check the new dependency with the people I mentioned

File: package.json (hunk 2 above)
Comment (jurokapsiar) [reply package.json:115]: @layershifter added this

Comment (layershifter) [reply package.json:115]: > can we suggest some existing tooling to achieve this?

Nope

> are we ok with adding this dependency?

Yep, it's only for docs

**Review by bmdalex — APPROVED** [review]

**Pull request merged**